rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,67 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+%global provider          github
+%global provider_tld      com
+%global project           percona-platform
+%global repo              dbaas-controller
+%global provider_prefix   %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path       %{provider_prefix}
+# commit value is dynamic, see build script:
+# https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+%global commit            0000000000000000000000000000000000000000
+%global shortcommit       %(c=%{commit}; echo ${c:0:7})
+%define build_timestamp   %(date -u +"%y%m%d%H%M")
+%define release           1
+%define rpm_release       %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
+
+%global install_golang    0
+
+Name:       %{repo}
+Version:    %{version}
```

Comment:
where %{version} is defined?

Reply:
As far as I can see, version is being `sed`'ed [here](https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm#L59).

Reply:
Let's change a value and add a comment about that like we did for `commit`

Reply:
Added comment with link to build script